src/main/resources/META-INF/resources/primefaces-extensions/fuzzysort/primefaces.fuzzysort.js

```diff
@@ -0,0 +1,52 @@
+PrimeFaces.widget.Fuzzysort = PrimeFaces.widget.BaseWidget.extend({
+
+    init: function (cfg) {
+        this.cfg = cfg;
+        this.id = this.cfg.id;
+        this.jqId = PrimeFaces.escapeClientId(this.id);
+//        this.jq = $(PrimeFaces.escapeClientId(this.cfg.target));
+
+        this.input = $(PrimeFaces.escapeClientId(this.cfg.id + "_fuzzysort-search-box"));
+        this.results = $(PrimeFaces.escapeClientId(this.cfg.id + "_fuzzysort-search-results"));
+        this.datasource = JSON.parse(this.cfg.value);
+
```

Comment:
You probably want to call `PrimeFaces.skinInput(this.input);` here which styles the Input with all the right PF styling.